app/code/core/Mage/Adminhtml/Block/Sales/Totals.php

```diff
@@ -56,8 +58,8 @@ protected function _initTotals()
         /**
          * Add shipping
          */
-        if (!$this->getSource()->getIsVirtual() && ((float) $this->getSource()->getShippingAmount() || $this->getSource()->getShippingDescription()))
-        {
+        if (!$this->getSource()->getIsVirtual()
+            && ((float) $this->getSource()->getShippingAmount() || $this->getSource()->getShippingDescription())) {
```

Comment:
```suggestion
        if (!$this->getSource()->getIsVirtual()
            && ((float) $this->getSource()->getShippingAmount() || $this->getSource()->getShippingDescription())
        ) {
```

Reply:
Style fixes were not planed ... but yes :+1: